homeassistant/components/roon/config_flow.py

```diff
@@ -6,19 +6,26 @@
 import voluptuous as vol
 
 from homeassistant import config_entries, core, exceptions
-from homeassistant.const import CONF_API_KEY, CONF_HOST
+from homeassistant.const import CONF_API_KEY, CONF_HOST, CONF_PORT
+import homeassistant.helpers.config_validation as cv
 
 from .const import (
     AUTHENTICATE_TIMEOUT,
     CONF_ROON_ID,
+    CONF_ROON_NAME,
     DEFAULT_NAME,
     DOMAIN,
     ROON_APPINFO,
 )
 
 _LOGGER = logging.getLogger(__name__)
 
-DATA_SCHEMA = vol.Schema({vol.Required("host"): str})
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("host", default="192.168.1.X"): cv.string,
```

Comment:
The default host ip address doesn't look valid? Is it valid?

Reply:
It’s just a example to suggest to the user, but ending in X isn’t valid. 

The first three parts are fairly typical, for a home network, but it’s hard to suggest a sensible final number.

Happy to change if there is an HA convention.

Reply:
If we don't know a default value that could work as is, I suggest not setting a default.

Reply:
Cool - now removed/